lib/rubygems/specification.rb

```diff
@@ -2239,7 +2239,7 @@ def original_platform # :nodoc:
   # The platform this gem runs on.  See Gem::Platform for details.
 
   def platform
-    @new_platform ||= Gem::Platform::RUBY
+    @new_platform ||= Gem::Platform::RUBY # rubocop:disable Naming/MemoizedInstanceVariableName
```

Comment:
There is already @platform used for different purpose. It could be refactored if needed (or later in other PR).

Reply:
I'm happy either way!